src/plugins/discover/public/application/main/components/chart/histogram.tsx

```diff
@@ -119,6 +124,31 @@ export function DiscoverHistogram({
     [dateFormat]
   );
 
+  const renderErrorMsgButton = () => {
+    return (
+      <EuiPanel color="transparent" hasBorder={false}>
+        <EuiButtonEmpty onClick={() => setShowError(!showError)}>
+          {showError ? (
+            <FormattedMessage
+              id="discover.errorLoadingChart.hideError"
+              defaultMessage="Hide error"
+            />
+          ) : (
+            <FormattedMessage
+              id="discover.errorLoadingChart.showError"
+              defaultMessage="Show error"
+            />
+          )}
+        </EuiButtonEmpty>
+        {showError && error ? (
+          <EuiCodeBlock language="json" paddingSize="s">
+            {error.message}
```

Comment:
Do you use the `EuiCodeBlock` intentionally here? since you are just showing `error.message` which AFAIK always a string?